client/landing/gutenboarding/components/signup-form/index.tsx

```diff
@@ -110,7 +110,7 @@ const SignupForm = ( { onRequestClose }: Props ) => {
 
 	const langFragment = lang ? `/${ lang }` : '';
 	const loginRedirectUrl = encodeURIComponent(
-		`${ window.location.origin }/${ GUTENBOARDING_BASE_NAME }${ makePath( Step.CreateSite ) }?new`
+		`/${ GUTENBOARDING_BASE_NAME }${ makePath( Step[ currentStep ] ) }`
```

Comment:
hmm in this change it no longer includes the window.location.origin, this causes local and calypso.live builds to take you to wordpress.com

Reply:
Thanks! I've just updated that one now.